modules/social_features/social_group/modules/social_group_invite/config/update/social_group_invite_update_13001.yml

```diff
@@ -0,0 +1,22 @@
+views.view.social_group_invitations:
```

Comment:
@volodymyr-sydor since we update this config in the hook update, shouldn't we also update `views.view.social_group_invitations.yml` ?

Reply:
@ribel Yes, already fixed it!